app-arch/innoextract/innoextract-1.5.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2013 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils toolchain-funcs cmake-utils
+
+DESCRIPTION="A tool to unpack installers created by Inno Setup"
+HOMEPAGE="http://constexpr.org/innoextract/"
+SRC_URI="http://constexpr.org/innoextract/files/${P}.tar.gz"
+
+LICENSE="ZLIB"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="c++0x debug doc +iconv +lzma static"
+
+RDEPEND="
+	!static? (
+		dev-libs/boost:=
+		iconv? ( virtual/libiconv )
+		lzma? ( app-arch/xz-utils )
+	)"
+DEPEND="${RDEPEND}
+	doc? ( >=app-doc/doxygen-1.8.3.1 )
+	static? (
+		app-arch/bzip2[static-libs]
+		dev-libs/boost[static-libs]
+		sys-libs/zlib[static-libs]
+		iconv? ( virtual/libiconv )
```

Comment:
Afaik `freebsd-lib` only provides `sys/iconv.h`, which is not what we want.

Aside from that, it seems dirty to hardcode all the different `libc`s when the virtual package is supposed to do that for us. Is it possible to add a static-libs use flag to it? On the other hand, dynamically linking `libiconv` doesn't seem too bad as it's also dynamically linked when using glibc's iconv.